ignite/utils.py

```diff
@@ -125,3 +126,38 @@ def setup_logger(
         logger.addHandler(fh)
 
     return logger
+
+
+def one_rank_only(*args, **kwargs):
```

Comment:
Please, specify `rank=0` as default signature.
For example, like that (https://github.com/pytorch/ignite/blob/master/ignite/metrics/metric.py#L296):
```python
def one_rank_only(rank=0)
    def wrapper(func):
        def another_wrapper(*args, **kwargs):
            if dist.get_rank() == rank:
                return func(*args, **kwargs)
    return wrapper
```
?

Maybe, we also need to use `functools.wraps`.

Reply:
Ok but api will be
```
@one_rank_only()
````
not
```
@one_rank_only   # no ()
```
with `rank` passed 
```
@one_rank_only(tank=...)
```

My hand pattern allow to do not have `()` and respect what is designed in issue. But it is less readable and more complicated for few benefits.

Reply:
I do not mind to have `@one_rank_only()` and simplified code :)

Reply:
Ok good ! 👍🏻